elife_bus_sdk/__init__.py

```diff
@@ -13,18 +13,18 @@
 PUBLISHER_TYPES = get_publisher_types()
 
 
-def get_publisher(pub_name: str = DEFAULT_NAME, pub_type: str = DEFAULT_TYPE,
-                  config: dict = {}) -> EventPublisher:
+def get_publisher(config: dict, pub_name: str = DEFAULT_NAME,
+                  pub_type: str = DEFAULT_TYPE) -> EventPublisher:
     """
     Publisher factory function.
 
     If a publisher already exists with the `pub_name` value
     then you will be returned that instance, otherwise the publisher
     will be created and returned.
 
+    :param config: dict
     :param pub_name: str
     :param pub_type: str
-    :param config: dict
```

Comment:
These are all redundant anyway?

Reply:
On the surface, it looks like they duplicate type hinting, but they allow the generation of documentation which you could then [host](https://readthedocs.org/). Even if it's not going to be used immediately it's nice to have, it's a habit but has a use.

Reply:
Hmm, sounds like a tooling issue.